_scripts/instruction-widget/install.js

```diff
@@ -98,16 +99,19 @@ module.exports = function(context) {
     // Debian Jessie
     context.base_command = "certbot";
     context.cron_included = true;
+    context.package = "certbot";
+
     if (context.webserver == "apache") {
       context.package = "python-certbot-apache";
-    } else {
-      context.package = "certbot"
+    } else if (context.webserver == "nginx") {
+      context.certonly = true;
```

Comment:
I would remove all of these `context.certonly = true`s
They make it so that the nginx instructions aren't shown.
Especially since we're recommending nginx as the installer, and there are no blocks in the nginx document that refer to certonly there's really no reason to set it inside of an nginx block.
Alternately you /could/ change the code in lines 11-27 of get-started.js, or maybe you should do both